src/components/FeedItem.vue

```diff
@@ -27,5 +27,11 @@ export default {
       return `https://www.invidio.us/watch?v=${this.entry.id}`;
     },
   },
+  filters: {
+    thousands_sep(number) {
+      // return number;
```

Comment:
Remove the leftover test comment, please!

Reply:
Resolved with 4c77b5a.